conode/conode.go

```diff
@@ -209,7 +210,7 @@ func checkConfig(c *cli.Context) error {
 		}
 	}
 
-	return nil
+	return check.CothorityCheck(tomlFileName, c.Bool("detail"))
```

Comment:
So this doubles up the behaviour of the blscosi check function (which is really kind of example code) with the "collect status from all nodes" behaviour @Gilthoniel put there before.

I think what we want is that the go.dedis.ch/cothority/blscosi service is imported into conode.go. Then if someone wants to use the blscosi client CLI (see blscosi/blscosi/blscosi.go) to run the check command, they can.

You should be sure that the test.sh in blscosi/blscosi still works after you make your changes. This is the integration test, and it could show you that something needed to be changed in the client side to make this change. test.sh is not run by travis, but is run by Jenkins.

Reply:
One of my goal was also to eject the key pair of this service from the public toml file. The question here is: is it worth to have it by default imported for the conodes or can we keep it optional ?

Reply:
* 9bff07d47ef88ca700363abf48a092fe8350b192 reverts to not-calling-by-default the check command. The package is imported however for anyone to manually run the check command.
* @Gilthoniel 's observation should be raised as a new issue.